package.json

```diff
@@ -14,7 +14,7 @@
 		"watch:pages": "cuttlebelle -w -n",
 		"watch:css": "onchange 'src/sass/**/*.scss' -- npm-run-all build:sass build:autoprefixer",
 		"watch:js": "onchange 'scripts/js/**/*.js' -- npm run build:js",
-		"watch:components": "onchange 'src/**/*.js' -- cuttlebelle docs",
+		"watch:components": "onchange 'src/**/*.js' -- npm run build:js",
```

Comment:
Please make a github issue referencing this change so you can fix it when prioritised

Reply:
https://github.com/govau/service-manual/issues/617